_includes/repository.html

```diff
@@ -24,7 +15,8 @@
         </div>
         <div id="progress">0%</div>
     </div>
-    </p>
-
     <div id="repository-releases">retrieving repository data...</div>
+    <div style="margin-top:10px">
+			<a href="users.html"><img src="css/logos.png"></a>
```

Comment:
s/<TAB>/<SPACE> ? I think someone recently updated the code to use tabs only?

Reply:
Arghh.  I hate tabs.  fixing...

Reply:
Done.